processor/processor.go

```diff
@@ -1390,15 +1390,9 @@ func (proc *HandleT) processJobsForDest(subJobs subJob, parsedEventList [][]type
 					// At the TP flow we are not having destination information, so adding it here.
 					shallowEventCopy.Metadata.DestinationID = destination.ID
 					shallowEventCopy.Metadata.DestinationType = destination.DestinationDefinition.Name
-
-					//TODO: Test for multiple workspaces ex: hosted data plane
-					/* Stream destinations does not need config in transformer. As the Kafka destination config
-					holds the ca-certificate and it depends on user input, it may happen that they provide entire
-					certificate chain. So, that will make the payload huge while sending a batch of events to transformer,
-					it may result into payload larger than accepted by transformer. So, discarding destination config from being
-					sent to transformer for such destination. */
-					if misc.Contains(customDestinations, *destType) {
-						shallowEventCopy.Destination.Config = nil
+					configsToFilter := destination.DestinationDefinition.ConfigFilters
```

Comment:
What happens to range operation if this is nil?

Reply:
`ConfigFilters` won't be `nil` as its type Array when initialized it will be an empty array. But we can do a nil check before iterating

Reply:
![Screenshot 2022-10-31 at 10 38 00 AM](https://user-images.githubusercontent.com/25389167/198935486-fb5ae93c-2388-4659-bfdf-c777d84f5342.png)
 Lint is marking it as not needed

Reply:
Yes looks like nil is handled by range so we don't need to worry about: https://play.golang.com/p/iymc4sfOdcw